x-pack/plugins/integration_assistant/server/integration_builder/build_integration.ts

```diff
@@ -27,9 +27,10 @@ export async function buildPackage(integration: Integration): Promise<Buffer> {
     autoescape: false,
   });
 
-  const tmpDir = joinPath(tmpdir(), `integration-assistant-${generateUniqueId()}`);
+  const workingDir = joinPath(getDataPath(), `integration-assistant-${generateUniqueId()}`);
```

Comment:
We could use the auto importer branding here

Reply:
@ilyannn I'll do in the next re-branding PR I am preparing